src/components/ContactDetails/ContactDetailsAvatar.vue

```diff
@@ -26,7 +26,7 @@
 		<div class="contact-header-avatar__wrapper">
 			<div class="contact-header-avatar__background" @click="toggleModal" />
 			<div v-if="contact.photo"
-				:style="{ 'backgroundImage': `url(${contact.photoUrl})` }"
+				:style="{ 'backgroundImage': `url(${contact.photoUrl})`, 'backgroundColor': 'rgba(255, 255, 255, 0.8)' }"
```

Comment:
Just use `#fff` here, since it looks weird with the color still shining through. :) Also we don’t need to theme it because with dark mode, the backgrounds of these images should still be white.

Reply:
I've changed it now, though I liked the old style a bit better. Made the contrast seem less harsh.

![SharedScreenshot](https://user-images.githubusercontent.com/1106400/72665991-feada480-3a0d-11ea-93cb-02e2c8e4c6ad.png)

Reply:
Could you also change it to white for the contact list? Because when there is an image, no color of our generator should shine through, as it has nothing to do with the actual image. :)